### ISSUE TITLE
Bug with Promethean 

### ISSUE THREAD

**Issue by matr0nix** [OpenBoard-org/OpenBoard#61]
Hello,

Sorry for my english.

I have a Promethean (and ActivBoard), and I work on Mac. I have installed all drivers I need.

There is a problem with the stylus. When I release the stylus and press it again, OpenBoard draws a line between these two points. So I can't write. 
I try SketchBook, and I haven't got the problem.

Can you solve it ? I can't use in classroom for now.

Thank you

**Comment by freeheel3r** [comment]
Hi,
Thanks for reporting this. Unfortunately without access to the hardware I really can't do anything about this, sorry. OpenBoard works fine with Wacom tablets and with every touch-based input we've tested, as far as I know. And that includes some interactive white boards (not sure which make & models though).

If someone has access to this kind of board and wants to try their hand at debugging this, please do :) I can always offer some advice if need be.

**Comment by SamuelLongchamps** [comment]
Follow-up on this? We have access to an ActivBoard and could not reproduce. Perhaps it's Mac-specific?

Does OpenBoard interact with lower level input at all? I assumed it used a generic mouse input provided by the OS like any other application.

**Comment by freeheel3r** [comment]
> I assumed it used a generic mouse input provided by the OS like any other application.

That's correct, the only thing OpenBoard knows is the input device's position, whether the (left) mouse button is pressed, and the pressure level (if there is one).

**Comment by SamuelLongchamps** [comment]
Well then, it seems like the issue is with the tool selection. If the pen is selected and not the line tool and you still have the issue, that would be a bug. If not, then it's a matter of usage.

To be confirmed for now (should we add this flag to issues? I see many issues without follow-up and a nice prioritization would be nice).

**Comment by freeheel3r** [comment]
I'm not sure it would be related to the tool selection. It sounds like OpenBoard (or Qt, or the OS) isn't getting the "mouse button released" information, which could be caused by anything from the hardware to the OS, or the driver (or OpenBoard).
The fact that it works with SketchBook would suggest a bug with OpenBoard, but I wouldn't be surprised if SketchBook has some lower-level code to improve compatibility with specific hardware.

Anyway yes tags would be good, I just haven't bothered with them much so far since I've been pretty much the only developer and I don't necessarily decide which issues to tackle.
What's a reasonable list of labels? I'm thinking;
- bug
- bug-unconfirmed
- enhancement
- support
- discussion

**Comment by matr0nix** [comment]
Hello,

It's look like the bug is resolved !

Now Openboard works great, thank you very much ! I use it everyday now.

**Comment by freeheel3r** [comment]
That is excellent news, thanks for letting us know! Is it a driver update that fixed this issue?
(Other people have reported this problem, so they might appreciate finding out how to fix it)

**Comment by madwax** [comment]
matr0nix,
Did you re-calibrate your ActivBoard?
